flatpak-import.py

```diff
@@ -0,0 +1,212 @@
+#!/usr/bin/env python2
+
+from argparse import ArgumentParser
+from ConfigParser import ConfigParser
+import gi
+gi.require_version('OSTree', '1.0')
```

Comment:
Generally this line and the import of gi preceede any real imports in files or they follow standard lib imports

```
import gi
gi.require_version('OSTree', '1.0')

import foo
import bar
import baz

from gi.repository import GLib, Gio, OSTree
```